app/src/main/java/com/example/navadon/androidonclickevent/Option0Activity.java

```diff
@@ -15,18 +17,39 @@
     private EditText etInput;
     private TextView tvOutput;
     private Button btnProcess;
+    private Button btnNext;
+    private Button btnBack;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_option0);
         bindView();
+        onClick();
+    }
+
+    private void onClick() {
+        btnNext.setOnClickListener(new View.OnClickListener() {
```

Comment:
It seems that you are not using the same approach as demonstrated. So I'm not sure if the instruction is clear enough. Please take a look at an example in assignment #3 that I've just added.